internal/database/actions.go

```diff
@@ -48,20 +106,18 @@ func GetResultTestKeys(sess sqlbuilder.Database, resultID int64) (string, error)
 		tk   PerformanceTestKeys
 	)
 	for res.Next(&msmt) {
-		if msmt.TestName == "web_connectivity" {
-			break
-		}
```

Comment:
Historically, why was this code here?

Reply:
I was under the impression I had done this to be equivalent, but I actually see there may be something different happening. I will take a look at this logic again when I go back to my editor.

Reply:
So I checked and actually given the comment I see, this is the correct behaviour. I suppose that if we start enriching that testKeys at this level for clients we may want to go back and change this, but for the moment I say it stays like this.

Note: the if statement below is more stringent than it was before.

Reply:
Gotcha!